### ISSUE TITLE
Add parameteriation of secondary ice

### ISSUE THREAD

**Issue by oyvindseland** [NorESMhub/CAM#152]
### What is the feature/what would you like to discuss?

 Parameterisation of secondary ice formation obtained from Georgia Sotiropoulou, developed from WRF and implemented in the Forces version of NorESM2. Changed files: micro_mg2_0.F90. New file module_random_forests.F90. Also changed namelist settings

### Will this change (regression test) answers?

Yes

### Will you be implementing this enhancement yourself?

Yes

**Comment by oyvindseland** [comment]
Should this be a default or an option in NorESM2.3? 
Given that is a CAM issue, not a Oslo-Aero issue this code must be rewritten for use in NorESM2.5  @DirkOlivie @MichaelSchulzMETNO

**Comment by MichaelSchulzMETNO** [comment]
I think the idea was, that this parameterisation can be activated with a namelist option, but not be included in "non forces" compsets.

**Comment by oyvindseland** [comment]
For code review and implementation is it best to include a namelist option in a first pull request or is it better to implement the code with a logical variable inside the relevant subroutine(s) ? @mvertens

**Comment by gold2718** [comment]
> For code review and implementation is it best to include a namelist option in a first pull request or is it better to implement the code with a logical variable inside the relevant subroutine(s) ? @mvertens

If you submit a PR indicating which logical variables should be namelist variables, I can very quickly submit a PR to your PR branch with those changes.

**Comment by oyvindseland** [comment]
For NorESM2.3 development. the Externals point to a CAM branch noresm2_3_develop
This branch however point to a Oslo-Aero tag 

For development should I first checkout the development branch (noresm2_3_develop) and create a branch from this or should I branch off from the tag, and then the new branch have to be merge with the development branch

**Comment by gold2718** [comment]
To develop the CAM code, I would:
```
git checkout -b my_cam_dev_branch origin/noresm2_3_develop
```
To develop the OSLO_AERO code I would do the same thing because the tag is behind the HEAD of noresm2_3_develop
```
git checkout -b my_oslo_aero_dev_branch origin/noresm2_3_develop 
```

**Comment by oyvindseland** [comment]
Opened pull request for the parameterisation in noresm2_3_develop

**Comment by gold2718** [comment]
Closed by #165
